.github/workflows/build.yml

```diff
@@ -60,6 +61,11 @@ jobs:
       with:
         path: ~/DF
         key: ${{ steps.env_setup.outputs.df_version }}
+    - name: Fetch ccache
+      uses: actions/cache@v2
+      with:
+        path: ~/.ccache
+        key: ${{ matrix.gcc }}
```

Comment:
Could this also include `${{ matrix.os }}`? That way we would use a clean cache if we move the GCC 7 build to a newer Ubuntu version, for instance.

More generally, I would tend towards more verbose keys to ensure that they are unique across all other caches in this pipeline, but that is probably not important for now. (One example: `ccache-${{ matrix.os }}-gcc-${{ matrix.gcc }}`)

Reply:
copy pasta'd